worlds/rabi_ribi/__init__.py

```diff
@@ -124,7 +124,8 @@ def generate_early(self) -> None:
         self.randomizer_data = RandomizerData(self.existing_randomizer_args)
 
         # Will be configurable later, but for now always force eggs to be local
-        self.options.local_items.value.add(ItemName.easter_egg)
+        if not self.options.allow_remote_easter_eggs:
```

Comment:
Remote eggs are not compatible with `encourage_eggs_in_late_spheres()`; we should throw an error if both are enabled.